pkgs/games/rbdoom-3-bfg/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, directx-shader-compiler
+, libGLU
+, openal
+, SDL2
+, vulkan-headers
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rbdoom-3-bfg";
+  version = "1.4.0";
+  src = fetchFromGitHub {
+    owner = "RobertBeckebans";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-r/dvTirgFXdBJ+Gjl6zpHoGCTPoo0tRmOCV9oCdnltI=";
+    fetchSubmodules = true;
+  };
+  nativeBuildInputs = [
+    cmake
+    directx-shader-compiler
+    libGLU
+    openal
+    SDL2
+    vulkan-headers
+  ];
```

```suggestion
  nativeBuildInputs = [
    cmake
  ];
  
  buildInputs = [
    directx-shader-compiler
    libGLU
    openal
    SDL2
    vulkan-headers
  ];
```

I think most of these are `buildInputs` rather than `nativeBuildInputs`. You can try building with `strictDeps = true` if you want to have Nix tell you when one is in the wrong input

---

I think `directx-shader-compiler` is only needed for `nativeBuildInputs`, but I've moved the others to `buildInputs`.